2025/selection-process.md

```diff
@@ -82,12 +82,12 @@ investigation efforts.
 
 ### Champion Selection
 
-**Timeline**: October 3rd, 10th, 17th
+**Timeline**: October 17th and 24th
```

Comment:
Why not 10th, 17th and 24th?

Reply:
10th was excluded to give some buffer, in case there is a late flurry of proposals and teams needed time to review. That said, since there is no expectation defined for the 10th, I don't see any harm in adding it back in.